src/meta/src/hummock/hummock_manager.rs

```diff
@@ -877,11 +870,11 @@ where
                 compact_status,
                 compact_task_assignment,
                 current_version_id,
-                hummock_versions,
                 hummock_version_deltas,
                 version_stale_sstables,
                 sstable_id_infos
             )?;
+            hummock_versions.commit();
```

Comment:
why moving this out from `commit_multi_var`?

Reply:
> why moving this out from `commit_multi_var`?

`commit_multi_var(hummock_versions)` has 2 parts:
- store `hummock_versions` into etc
- `hummock_versions.commit();`

here we remove first part

Reply:
I see. Please add some documentations along with future PRs then.

Reply:
> I see. Please add some documentations along with future PRs then.

ok